selene/common/helpers.py

```diff
@@ -39,18 +44,27 @@ def to_by(selector_or_by: Union[str, tuple]) -> Tuple[str, str]:
         return selector_or_by
     if isinstance(selector_or_by, str):
         return (
-            (By.XPATH, selector_or_by) if (
+            (By.XPATH, selector_or_by)
+            if (
```

Comment:
if, and, or operators if the beginning of new line - looks more readable than in the end of previous line - "black" keep an eye on this it too and helps to format.